src/commands/Project.ts

```diff
@@ -60,6 +62,38 @@ export class Project extends Command {
         }
       }
     },
+    debug: {
+      when: () => {
+        return RxFs.exist(path.join(process.cwd(), 'relution.hjson'));
+      },
+      why: () => {
+        return this.i18n.DEBUGGER_OPEN_WHY;
+      },
+      description: this.i18n.DEBUGGER_OPEN_DESCRIPTION,
+      vars: {
+        server: {
+          pos: 0
+        }
+      }
+    },
+    log: {
+      label: 'logger',
```

Comment:
I wonder why the function is named log instead of logger like the menu item is named. Is there a reason for this?